### PR TITLE
tul/merge-into-clarin-dspace (#316)

### LINKED ISSUE #392: internal/merge TUL addons into CLARIN-DSpace7.5
## Problem description
The code updates from TUL customer customization merge into CLARIN-DSPace7.5. codebase.

PR BE: https://github.com/dataquest-dev/DSpace/pull/402
PR FE: https://github.com/dataquest-dev/dspace-angular/pull/317

### PR DESCRIPTION
**Pull request by milanmajchrak**
## Problem description
Add TUL updates into CLARIN-DSpace7.5.

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -88,7 +88,7 @@ jobs:
           labels: ${{ steps.meta_build.outputs.labels }}
 
       - name: redeploy
-        if: 'false'
+        if: '!cancelled()'
```
Comment (MajoBerger) [inline .github/workflows/docker.yml]: should be false, not !cancelled()

File: .github/workflows/erase_db.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ jobs:
         run: |
           docker volume rm dspace-${{env.INSTANCE}}_pgdata dspace-${{env.INSTANCE}}_assetstore
       - name: redeploy
-        if: 'false'
+        if: '!cancelled()'
```
Comment (MajoBerger) [inline .github/workflows/erase_db.yml]: as above, should be false, not !cancelled()

File: src/app/contact-page/contact-page.component.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.email-text{
+    margin-left: 5rem;
+}
```
Comment (MajoBerger) [inline src/app/contact-page/contact-page.component.scss]: missing end line

File: src/app/item-page/simple/field-components/specific-field/citation/item-page-citation.component.html [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<iframe
+  *ngIf ="citaceProStatus"
+  height="200px"
+  width="100%"
+  frameborder="0"
+  [src]="iframeSrc"
+></iframe>
```
Comment (MajoBerger) [inline src/app/item-page/simple/field-components/specific-field/citation/item-page-citation.component.html]: missing end line

**Review by MajoBerger — CHANGES_REQUESTED** [review]

File: .github/workflows/docker.yml (hunk 1 above)
Comment (milanmajchrak) [reply .github/workflows/docker.yml]: fixed

File: .github/workflows/erase_db.yml (hunk 2 above)
Comment (milanmajchrak) [reply .github/workflows/erase_db.yml]: fixed

**Review by Paurikova2 — APPROVED** [review]

**Pull request merged**